### PR TITLE
Misc queue from 2018 (sorry, lost sources)

### PR DESCRIPTION
**Pull request by Gelma**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -10176,6 +10177,7 @@ overwritren->overwritten
 overwrittes->overwrites
 overwrittin->overwriting
 overwritting->overwriting
+ovrridables->overridables
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: Can you add the singular please

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -11286,6 +11294,7 @@ propertly->properly, property,
 propertys->properties
 propertyst->properties
 propery->property, properly,
+properyvalue->propertyvalue
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt]: This seems specific.

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Done

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Removed

**Comment by larsoner** [comment]
Whenever you're happy feel free to merge. I think one set of eyes on each PR is probably good enough

**Comment by peternewman** [comment]
> Whenever you're happy feel free to merge. I think one set of eyes on each PR is probably good enough

I can't guarantee I won't miss stuff/someone else won't spot other bits, but I guess we ought to be practical to get through this backlog, and users will presumably find it in the real world. 

Is this configured in GitHub too then, that PRs need to pass Travis and Appveyor and have an approved review?

**Comment by larsoner** [comment]
No it's not configured that way, I always just look. But it might be possible to add this step if you want

**Comment by peternewman** [comment]
> No it's not configured that way, I always just look. But it might be possible to add this step if you want

Yeah you can via protected branches ( https://help.github.com/en/articles/defining-the-mergeability-of-pull-requests ), we use it in another repo I work on, even for admins, and while it can occasionally be a small faff (and I've turned it off briefly for admins once or twice to get something through), I'd say it's generally a good idea. I think it also gives some UI hints to contributors to tell them Travis isn't working and it can't be merged.

**Comment by larsoner** [comment]
Okay, I think I added the protection

**Comment by peternewman** [comment]
> Okay, I think I added the protection

It looks like statuses are required, but not approving reviews, see e.g. this one: https://github.com/codespell-project/codespell/pull/1134

**Comment by larsoner** [comment]
I didn't enable that one because it seems like unnecessary burden (extra clicks to start and approve a review) given the unlikelihood of errant merge

**Comment by peternewman** [comment]
> I didn't enable that one because it seems like unnecessary burden (extra clicks to start and approve a review) given the unlikelihood of errant merge

Okay, fair enough.

**Comment by peternewman** [comment]
Did you also tick "Require branches to be up to date before merging", we get a button to update on another repo, and will persuade people to resync?

**Comment by peternewman** [comment]
Actually that may be irrelevant apart from new dictionary tests, as conflicts will still block.

**Comment by larsoner** [comment]
Travis builds the merged version of each PR so it shouldn't be necessary

**Pull request merged**